ci-operator/step-registry/ipi/conf/aws/launcher-info/ipi-conf-aws-launcher-info-commands.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+#set -o nounset
+#set -o errexit
+#set -o pipefail
+
+set +e
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+cp ${CLUSTER_PROFILE_DIR}/pull-secret /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_LATEST} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
+ocp_major_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $1}' )
+ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+rm /tmp/pull-secret
+
+job_type="${JOB_TYPE:-}"
+user=""
+pull_number=""
+if [[ "${job_type}" == "presubmit" ]]; then
+    user=$(echo "${JOB_SPEC:-}" | jq -r '.refs.pulls[].author')
+    pull_number=${PULL_NUMBER:-unknow}
```

Comment:
typo for `unknow`....should be `unknown`

Reply:
Done